launcher.py

```diff
@@ -46,6 +46,12 @@ def get_parser_args():
         type=int,
         help="Number of CPU cores to use in this pipeline run (default %(default)s)")
 
+    parser.add_argument(
+        "--num-batches",
+        default=1,
+        type=int,
+        help="Number of batches to split input vcf.gz file into (default is 1)")
```

Comment:
Add "Splitting can speed up the preprocessing stage for the large datasets."

Reply:
Thank you, done